Update screenshots and videos for experiments, feature-flags, hogql, notebooks session-replys, surveys, data, web-analytics, toolbar, and select product-analytics articles in docs for Issue 7616 (#7907)
PostHog/posthog.com: contents/docs/feature-flags/creating-feature-flags.mdx

    selfServe: full
    enterprise: full
---
import { ProductScreenshot } from 'components/ProductScreenshot'
import CreateNewFeatureFlagLight from '../../images/features/feature-flags/create-new-feature-flag-light-mode.png'
import CreateNewFeatureFlagDark from '../../images/features/feature-flags/create-new-feature-flag-dark-mode.png'
import ReleaseConditionLight from '../../images/features/feature-flags/release-conditions-light-mode.png'
import ReleaseConditionDark from '../../images/features/feature-flags/release-conditions-dark-mode.png'

<iframe
    src="https://www.youtube-nocookie.com/embed/2jQco8hEvTI?start=774"
    className="rounded shadow-xl"
/>

> To create a new feature flag, navigate to the [feature flags tab](https://app.posthog.com/feature_flags) in your PostHog app, where you should see a 'Create feature flag' button that we'll click.

Reviewer: Same comment as above, they might have one already 🤷 
```suggestion
To create a new feature flag, navigate to the [feature flags tab](https://app.posthog.com/feature_flags) in your PostHog app, and click the "New feature flag" button in the top right.
```
